content/Building an Engine-modularized API in Rails - Guillermo Leopold.md

```diff
@@ -0,0 +1,154 @@
+# Building an Engine-modularized API in Rails
+
+![main image](images/engines_main_image.jpeg)
+
+When big development teams build an API, one of the main challenges that we face is being able to work together on different features while avoiding bottle-necks or other common issues such as conflicting perspectives or avoid overlapping efforts on particular features.
+
+Because of this, when we started working on our latest project, we decided to work with a new approach; designed to help us to scale by splitting the app into microservices.
```

Comment:
About this, you could add a reference or recommendation of this article:
https://martinfowler.com/bliki/MonolithFirst.html
it's about this approach we took, of modularizing an app instead of starting with microservices.